codec-http/src/main/java/io/netty/handler/codec/http/websocketx/WebSocketClientHandshaker.java

```diff
@@ -79,14 +88,18 @@
      *            Map of custom headers to add to the client request
      * @param maxFramePayloadLength
      *            Maximum length of a frame's payload
+     * @param forceCloseTimeoutMillis
+     *            Close the connection if it was not closed by the server after timeout specified
      */
     protected WebSocketClientHandshaker(URI uri, WebSocketVersion version, String subprotocol,
-                                        HttpHeaders customHeaders, int maxFramePayloadLength) {
+                                        HttpHeaders customHeaders, int maxFramePayloadLength,
+                                        long forceCloseTimeoutMillis) {
```

Comment:
as this is protected we will need to also retain the old constructor as otherwise its API breakage:

```
20:53:51 [ERROR] Failed to execute goal com.github.siom79.japicmp:japicmp-maven-plugin:0.13.1:cmp (default) on project netty-codec-http: There is at least one incompatibility: io.netty.handler.codec.http.websocketx.WebSocketClientHandshaker.WebSocketClientHandshaker(java.net.URI,io.netty.handler.codec.http.websocketx.WebSocketVersion,java.lang.String,io.netty.handler.codec.http.HttpHeaders,int):CONSTRUCTOR_REMOVED -> [Help 1]
```

Reply:
Done!